apps/astarte_appengine_api/test/astarte_appengine_api/device/device_test.exs

```diff
@@ -153,9 +153,16 @@ defmodule Astarte.AppEngine.API.DeviceTest do
     DatabaseTestHelper.seed_data()
   end
 
+  setup :verify_on_exit!
+
   setup_all do
     {:ok, _client} = DatabaseTestHelper.create_test_keyspace()
 
+    Mox.stub_with(
+      Astarte.AppEngine.API.RPC.VMQPlugin.ClientMock,
+      Astarte.AppEngine.API.RPC.VMQPlugin.Client
+    )
```

Comment:
Do we really need to stub the mock with the actual module implementation?

Reply:
No ! fixing :+1: